menas/src/main/resources/start_menas.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+echo "$PRIVATE_KEY" >> conf/private.pem
```

Comment:
Change "$variable" to "${variable}"
Just a better convention, no functional difference here but is helpful in cases where debugging may be required